index.html

```diff
@@ -1,16 +1,72 @@
 <!DOCTYPE html>
 <html>
+    
+    <input type="text" id="mySearch" onkeyup="myFunction()" placeholder="Search.." title="Type in a category">
+
+<ul id="myMenu">
+  <li><a href="file:///Users/osvaldoescobar/Projects/Thinkment/philosophical_issues.html">Philosophical Issues</a></li>
```

Comment:
Don't link to the location on your computer. Remove all of this: "file:///Users/osvaldoescobar/Projects/Thinkment/" so that it will simply look in the same folder as index.html. It will treat it as a sibling that way and just look for the files next to it.